bitnami/mxnet/templates/NOTES.txt

```diff
@@ -0,0 +1,51 @@
+{{- if and (contains "bitnami/" .Values.image.repository) (not (.Values.image.tag | toString | regexFind "-r\\d+$|sha256:")) }}
```

Comment:
It is also included at the end of the file

Reply:
Fixed